scimono-server/src/main/java/com/sap/scimono/entity/bulk/validation/BulkOperationsValidator.java

```diff
@@ -89,8 +89,10 @@ public List<RequestOperation> getValidBulkOperations(BulkBody<RequestOperation>
   }
 
   public BulkBody<ResponseOperation> getValidResponseData(BulkBody<RequestOperation> bulkRequest, BulkBody<ResponseOperation> bulkResponse) {
```

Comment:
This method has several other issues:
1. It breaks the SRP in the class. If the class is named `BulkOperationsValidator` then it should only validation. However, the `getValidResponseData` method has nothing to do with validation because it only assigns `location` to each `ResponseOpeartion`. Therefore, the method should be moved to another (new) class `BulkResponseOperationLocationService`.
2. The method's name does not reflect what is does: `getValidResponseData(BulkBody<ResponseOperation> bulkResponse)` -> `rebuildWithLocations(BulkBody<ResponseOperation> bulkResponse)`